src/components/HeaderDateRangeSelect.tsx

```diff
@@ -10,23 +10,27 @@ import {
 
 interface Props {
   exploreUrl?: ExploreUrl;
+  setDateRangeSelector?: any;
 }
 
-export const HeaderDateRangeSelect = ({ exploreUrl }: Props) => {
-  if (!exploreUrl) {
-    return null;
-  }
+export const HeaderDateRangeSelect = ({ exploreUrl, setDateRangeSelector }: Props) => {
+  const [dateRangeValue, setDateRangeValue] = useState<string>('Past6M');
 
   const handleChange = (event: React.ChangeEvent<{ value: unknown }>) => {
-    if (isDateRangeEncoded(event.target.value)) {
+    if (isDateRangeEncoded(event.target.value) && exploreUrl) {
       exploreUrl.setDateRange(dateRangeUrlToSelector(event.target.value));
     }
+    if (setDateRangeSelector && isDateRangeEncoded(event.target.value)) {
+      setDateRangeValue(event.target.value);
+      setDateRangeSelector(dateRangeUrlToSelector(event.target.value));
+    }
```

Comment:
It would be nicer to nest the `if`s:

```
if (isDateRangeEncoded(event.target.value)) {
  if (exploreUrl) {...}
  if (setDateRangeSelector) {...}
}
```